x-pack/test/api_integration/apis/transform/transforms_nodes.ts

```diff
@@ -20,14 +20,14 @@ export default ({ getService }: FtrProviderContext) => {
 
   const expected = {
     apiTransformTransformsNodes: {
-      count: 1,
+      minCount: 1,
     },
   };
 
   function assertTransformsNodesResponseBody(body: GetTransformNodesResponseSchema) {
     expect(isGetTransformNodesResponseSchema(body)).to.eql(true);
 
-    expect(body.count).to.eql(expected.apiTransformTransformsNodes.count);
+    expect(body.count).to.not.be.lessThan(expected.apiTransformTransformsNodes.minCount);
```

Comment:
The `expect` doesn't offer a `greater than or equal` assertion, so I had to work around it with `not less than`, which has the same effect.